### PR TITLE
Backports 0.13.1

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 ## Release History
 
+0.13.1 (...):
```
Comment (haampie) [inline README.md:102]:
```suggestion
0.13.1 (November 27, 2021):
```

**Review by haampie — COMMENTED** [review]
The release date has to be change, and I'm not sure if any other build files have to be updated, as I'm not a nix user

**Comment by Mic92** [comment]
> The release date has to be change, and I'm not sure if any other build files have to be updated, as I'm not a nix user

No. the version should be the only relevant file.

**Comment by Mic92** [comment]
merged in https://github.com/NixOS/patchelf/commit/1b4890a658e881e4f1d4b3c0a49577d8bae03075

**Comment by Mic92** [comment]
Thanks. I tagged the release.

**Pull request closed without merging**